src/services/comment.service.ts

```diff
@@ -1,13 +1,18 @@
 import { CommentEntity } from '@/entities';
-import { BaseCommentDto } from '@/dtos';
+import { BaseCommentDto, CreateCommentRequestDto } from '@/dtos';
 import { HttpException } from '@/exceptions';
 
-class CommentService{
-    public async getComments(): Promise<BaseCommentDto[]> {
-        const comments: BaseCommentDto[] = await CommentEntity.find()
-        if (comments.length === 0) throw new HttpException(404, 'No Comments Found')
-        return comments
-    }
+class CommentService {
+  public async getComments(): Promise<BaseCommentDto[]> {
+    const comments: BaseCommentDto[] = await CommentEntity.find();
+    if (comments.length === 0) throw new HttpException(404, 'No Comments Found');
+    return comments;
+  }
+
+  public async postCommentInDB(commentData: CreateCommentRequestDto): Promise<BaseCommentDto> {
```

Comment:
nit: I personally would just name this postComment. Up to you if you want to change it.